sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -497,3 +540,152 @@ class UpdateMode(str, Enum):
 class SASProtocol(str, Enum):
     HTTPS = "https"
     HTTP = "http"
+
+
+class PartialBatchErrorException(HttpResponseError):
```

Comment:
I don't think this should be in `_models`, i like to reserve this file for only the models we expose to users

Reply:
no opinion where this goes, but this is exposed to users

Reply:
@iscai-msft This is part of the batching operations that haven't been built yet, it will be exposed to users eventually

Reply:
ok nvm me